5.sc_Morphology_Logistic_Regression/1.train_models/0.train_single_class.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    data_df = pq.read_table(data_path).to_pandas()
</pre><p>Is this quicker than pd.read_parquet?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>probably not, I need to switch these over to <code>pd.read_parquet</code></p><p> </p>